client/influxdb.go

```diff
@@ -124,11 +124,15 @@ func (c *Client) Write(bp BatchPoints) (*Response, error) {
 
 	var b bytes.Buffer
 	for _, p := range bp.Points {
-		if p.Raw != "" {
+		if p.Raw != "" && len(bp.Tags) == 0 {
```

Comment:
Can you remove that line?  

Seems a different fix for `Raw` lines is needed and I'm not sure that the client should really have a `Raw` field to begin with.